modules/logging/logging.go

```diff
@@ -35,9 +37,21 @@ func SharedInstance() *LogStruct {
 	return sharedInstance
 }
 
-func (u *LogStruct) MethodInfo(model string, method string) *logrus.Entry {
-	return u.log.WithFields(logrus.Fields{
-		"model":  model,
-		"method": method,
-	})
+func (u *LogStruct) MethodInfo(model string, method string, stack ...bool) *logrus.Entry {
```

Comment:
__[golint]__
* [golint] exported method LogStruct.MethodInfo should have comment or be unexported